notification/telegram.py

```diff
@@ -1,14 +1,29 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
-#
+
 # (c) 2016, Artem Feofanov <artem.feofanov@gmail.com>
 #
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
 
 
 DOCUMENTATION = """
 
 module: telegram
-version_added: "0.1"
+version_added: "2.1"
```

Comment:
The door has been closed on v2.1 that you'll need to update this to at least 2.2 for Travis to start passing again.